Update GCP monitoring and logging (#94)
ansible/aap-docs: stories/topics/ref-aap-gcp-configure-google-cloud-ops-agent.adoc

The configuration file details can be found link:https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration[here].

The configuration file is located in `/etc/google-cloud-ops-agent/config.yml`.

This is a symbolic link to the shared disk `/aap/bootstrap/config_file_templates/controller/gcp-ops-agent-config.yml` or `/aap/bootstrap/config_file_templates/hub/gcp-ops-agent-config.yml` depending on the component type.

The configuration file contains a number of receivers specifying what should be collected by the ops agent.

Your selection of *Connect Logging* and *Connect Metrics* during deployment determines which pipelines are included in the file and therefore which logs and metrics are collected and sent to GCP.

If you need to add more pipelines post-deployment, you can insert them in `/aap/bootstrap/config_file_templates/hub|controller/gcp-ops-agent-config.yml`.

> A crontab job restarts the agent if `gcp-ops-agent-config.yml` changed in the last 10 minutes and thus the agent rereads its configuration.

Reviewer: ```suggestion
A crontab job restarts the agent if `gcp-ops-agent-config.yml` changed in the last 10 minutes. The agent rereads its configuration after a restart.
```
